github/ci/prow-deploy/kustom/base/configs/current/orgs/orgs.yaml

```diff
@@ -193,7 +193,7 @@ orgs:
         has_projects: false
         has_wiki: false
       dra-pci-driver:
-        allow_merge_commit: false
+        allow_merge_commit: true
```

Comment:
@xpivarc @alicefr Hi, this is the default merge method, so there is no need to set it explicitly. Unless I am missing something

Reply:
Relying on this [line](https://github.com/kubernetes-sigs/prow/blob/6925e9af79a44bf82b2c4d8c5fe7e125f3818969/pkg/config/tide.go#L459)

Reply:
Ok I removed the last commit

Reply:
@enp0s3 
Here we are setting the repository, if you keep this set to false than default merge method is not allowed! Please see note from tide https://github.com/kubevirt/dra-pci-driver/pull/1
@alicefr Please make a PR with this change

Reply:
@xpivarc @alicefr My bad. My intent was to remove the explicit false, because IIUC by default its true.

Reply:
we probably need to add it explicitly, it was set as default when I removed the option

Reply:
@alicefr This is super strange, I see the kubevirt repo configuration for example:
```
      kubevirt:
        allow_rebase_merge: false
        allow_squash_merge: false
        default_branch: main
        description: Kubernetes Virtualization API and runtime in order to define and manage virtual machines.
        has_projects: true
        homepage: https://kubevirt.io
```
It doesn't have `allow_merge_commit: true` set explicitly, nevertheless AFAIK we do see commits with "merge.." in the content in the main branch